js/constants.js

```diff
@@ -261,6 +261,5 @@ export const JOIN_POST_URL = `${BASE_URL}/users/self/intro`;
 export const SELF_URL = `${BASE_URL}/users/self`;
 export const GITHUB_OAUTH =
   'https://github.com/login/oauth/authorize?client_id=23c78f66ab7964e5ef97';
-export const SIGNUP_URL =
-  'https://my.realdevsquad.com/new-signup';
+export const SIGNUP_URL = 'https://my.realdevsquad.com/new-signup';
```

Comment:
can you revert it ?

Reply:
it happen during fixing lint not able to revert it

Reply:
https://dev.to/lofiandcode/git-and-github-how-to-revert-a-single-file-dha

Reply:
when i am running npm run fix it will change to this...not resolved

Reply:
wait, but what is wrong? other than the formatting?

Reply:
nothing

Reply:
exactly